Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityConfigurationProfileInspector.cs

```diff
@@ -15,8 +15,8 @@ public class MixedRealityConfigurationProfileInspector : MixedRealityBaseConfigu
     {
         private static readonly GUIContent TargetScaleContent = new GUIContent("Target Scale:");
         private static readonly GUIContent NewProfileContent = new GUIContent("+", "Create New Profile");
-        private static readonly GUIContent PlatformRenderingContent = new GUIContent("Platform Rendering:");
-        private static readonly GUIContent BoundaryHeightContent = new GUIContent("Boundary Height (in m):");
+        private static readonly GUIContent BoundaryHeightContent = new GUIContent("Boundary Height:");
+        private static readonly GUIContent BoundaryVisualizationProfileContent = new GUIContent("Boundary Visualization Profile");
```

Comment:
Why create a gui content for the auto generated content that the inspector will render by default?

Reply:
was following the pattern set by other inspector fields.. I can remove